setting_test.go

```diff
@@ -154,6 +168,20 @@ func TestSetting(t *testing.T) {
 			expected: []string{"one", "two", "three"},
 			bad:      make(map[string]interface{}),
 		},
+		{
+			name:     "StringMapStringSlice from JSON",
+			setting:  NewStringMapStringSliceSetting("StringMapStringSlice", "", nil),
+			good:     `{"animals": ["cats", "dogs", "fish"]}`,
```

Comment:
suggestion: Similar to my other suggestion, I recommend you add another key in this map to ensure that we actually load more than one key.